### PR TITLE
[Nginx] Add Missing ECS Field Mappings

### LINKED ISSUE #6917: [Nginx] Missing ECS Field Mappings
## Describe the Issue

Our `logs-*` Data View has conflicts for `related.ip`, `source.geo.dma_code`, and `source.geo.post_code`. 

All other Data Streams:
`related.ip` = field type `ip`
`source.geo.dma_code` = field type `keyword`
`source.geo.postal_code` = field type `keyword`

Nginx:
`related.ip` = field type `keyword`
`source.geo.dma_code` = field type `long`
`source.geo.postal_code` = field type `long`

I checked, and `ecs.yml` does not have mappings for any of these three fields.

### PR DESCRIPTION
**Pull request by MakoWish**
## Type of change
- Bug

## What does this PR do?

This PR corrects a field mapping conflict in the `logs-*` Data View from Nginx incorrectly mapping `source.geo.dma_code`, `source.geo.postal_code`, and `related.ip`. 

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have incremented the version in my package's `manifest.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #6917 

## Screenshots

conflicts
related ip
source geo dma_code
source geo postal_code

### CODE REVIEW

File: packages/nginx/data_stream/access/fields/ecs.yml [hunk 1]
```diff
@@ -1,5 +1,33 @@
+- external: ecs
+  name: destination.address
+- external: ecs
+  name: destination.as.number
+- external: ecs
+  name: destination.as.organization.name
 - external: ecs
   name: destination.domain
+- external: ecs
+  name: destination.geo.city_name
```
Comment (lalit-satapathy) [inline packages/nginx/data_stream/access/fields/ecs.yml:10]: Do we need to add these mapping for destination.geo.*, as I dont see these fields being populated somewhere in nginx.

**Review by lalit-satapathy — COMMENTED** [review]
Thanks for filing the PR!

File: packages/nginx/data_stream/access/fields/ecs.yml (hunk 1 above)
Comment (MakoWish) [reply packages/nginx/data_stream/access/fields/ecs.yml:10]: They are not populated by Nginx, but any customer that wants to enrich the data with Geo Location will require these fields to be mapped.

**Pull request closed without merging**